libs/pulumi-static-site/src/cloudfront.ts

```diff
@@ -102,20 +99,15 @@ export class Distribution extends pulumi.ComponentResource {
                     minimumProtocolVersion: 'TLSv1.2_2019',
                     sslSupportMethod: 'sni-only',
                 },
-                webAclId:
-                    args.webAclId ??
-                    pulumi.output(
-                        aws.waf.getWebAcl(
-                            { name: defaultManagedWebAcl },
-                            { parent: this },
-                        ),
-                    ).id,
                 comment: pulumi.interpolate`Static Site Distribution for ${pulumi
                     .output(args.domains)
                     .apply((domains) => domains.join(', '))}`,
                 tags: args.getTags(name),
             },
-            { parent: this },
+            {
+                parent: this,
+                ignoreChanges: [ 'webAclId' ],
```

Comment:
Maybe add `distributionIgnoreChanges` to `opts` instead. This would trip anyone else using the module up.